common/locales/en/faq.json

```diff
@@ -23,27 +23,27 @@
-  "androidFaqAnswer6": "At level 3, you will unlock the Drop System. Every time you complete a task, you'll have a random chance at receiving an egg, a hatching potion, or a piece of food. They will be stored in Menu > Items.\n\n To hatch a Pet, you'll need an egg and a hatching potion. Tap on the egg to determine the species you want to hatch, and select \"Hatch Egg.\" Then choose a hatching potion to determine its color! Go to Menu > Pets to equip your new Pet to your avatar by clicking on it. \n\n You can also grow your Pets into Mounts by feeding them under Menu > Pets. Tap on a Pet, and then select \"Feed Pet\"! You'll have to feed a pet many times before it becomes a Mount, but if you can figure out its favorite food, it will grow more quickly. Use trial and error, or [see the spoilers here](http://habitica.wikia.com/wiki/Food#Food_Preferences). Once you have a Mount, go to Menu > Mounts and tap on it to equip it to your avatar.\n\n You can also get eggs for Quest Pets by completing certain Quests. (See below to learn more about Quests.)",
+  "androidFaqAnswer6": "At level 3, you will unlock the Drop System. Every time you complete a task, you'll have a random chance at receiving an egg, a hatching potion, or a piece of food. They will be stored in Menu > Items.\n\n To hatch a Pet, you'll need an egg and a hatching potion. Tap on the egg to determine the species you want to hatch, and select \"Hatch with potion.\" Then choose a hatching potion to determine its color! Go to Menu > Stable > Pets > Pets Category > Pet to equip your new Pet to your avatar by clicking on it and then selecting use(Your avatar doesn't update to reflect the change). \n\n You can also grow your Pets into Mounts by feeding them under Menu > Stable > Pets. Tap on a Pet, and then select \"Feed\"! You'll have to feed a pet many times before it becomes a Mount, but if you can figure out its favorite food, it will grow more quickly. Use trial and error, or [see the spoilers here](http://habitica.wikia.com/wiki/Food#Food_Preferences). Once you have a Mount, go to Menu > Stable > Mounts > Mounts Category > Mount and tap on it and click use to equip it to your avatar(Your avatar doesn't update to reflect the change).\n\n You can also get eggs for Quest Pets by completing certain Quests. (See below to learn more about Quests.)",
```

Comment:
**`Go to Menu > Stable > Pets > Pets Category > Pet`**

This is misleading because "Pets Category" and "Pet" are not textual menu items that you select.

**`equip your new Pet to your avatar by clicking on it and then selecting use(Your avatar doesn't update to reflect the change).`**

"Use" should be quoted and capitalized in the same way as the other button text.  Also, not sure if it's necessary to note that your avatar may not reflect the change.  And if we do note it, this is an awkward way to do it.

**`Once you have a Mount, go to Menu > Stable > Mounts > Mounts Category > Mount`**

Same problem as above.

**`click use to equip it to your avatar(Your avatar doesn't update to reflect the change).`**

Same problems as above.